pkgs/by-name/tp/tpm2-pkcs11/package.nix

```
@@ -15,48 +16,59 @@
   python3,
   glibc,
   libyaml,
+  cmocka,
   abrmdSupport ? true,
```

I personally would prefer to disable user‐resource manager support by default in favour of the kernel‐space one. You can ensure that the other code paths get tested regularly with a passthrough test.

```suggestion
  abrmdSupport ? false,
```

---

Agreed, the kernel resource manager is better.